tests/e2e/manifests/chi/test-037-1-storagemanagement-switch.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: "clickhouse.altinity.com/v1"
+kind: "ClickHouseInstallation"
+metadata:
+  name: "test-037-storagemanagement-switch"
+spec:
+  useTemplates:
+    - name: clickhouse-version
+  configuration:
+    zookeeper:
```

Comment:
Remove ZooKeeper and convert to a single replica test to make test faster. You are testing 0-0-0 node anyway.